### PR TITLE
SD-3739: Include pipeline IdName while fetching workflow if applicable

### PR DESCRIPTION
**Pull request by abhishek-bhuskute**
- Resolve pipeline-name while fetching an edit-property action

### CODE REVIEW

File: src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "actions": [
+    {
+      "id": "418b1d4c-c7de-49fd-851f-0503d96830b7",
+      "type": "EDIT_PROPERTY",
+      "metaData": {
+        "pipeline": {
+          "id": 122,
+          "name": "Demo"
+        }
+      },
```
Comment (alpeshjikadra) [inline src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json:22]: This must not be metadata anymore

File: src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      "payload": {
+        "name": "pipeline",
+        "value": "1",
```
Comment (alpeshjikadra) [inline src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json:25]: And this must be an object IdName

File: src/test/resources/contracts/workflow/api/update-workflow-pipeline-request.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "conditionType": "FOR_ALL",
+    "values": []
+  },
+  "actions": [
+    {
+      "id": "08518b20-23f9-11eb-adc1-0242ac120002",
+      "type": "EDIT_PROPERTY",
+      "payload": {
+        "name": "pipeline",
+        "value": 1,
```
Comment (alpeshjikadra) [inline src/test/resources/contracts/workflow/api/update-workflow-pipeline-request.json:21]: This also be an IdName right ?

**Review by alpeshjikadra — CHANGES_REQUESTED** [review]

File: src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json (hunk 1 above)
Comment (abhishek-bhuskute) [reply src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json]: This file is not being used anymore. I had added a new one. Removed this.

File: src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json (hunk 2 above)
Comment (abhishek-bhuskute) [reply src/test/resources/contracts/workflow/api/integration/workflow-301-update-with-pipeline-edit.json]: Same goes for this.

File: src/test/resources/contracts/workflow/api/update-workflow-pipeline-request.json (hunk 3 above)
Comment (abhishek-bhuskute) [reply src/test/resources/contracts/workflow/api/update-workflow-pipeline-request.json]: Right. This one is not being used. Deleted these.

**Review by bhaskarpawar123 — APPROVED** [review]

**Review by alpeshjikadra — APPROVED** [review]

**Pull request merged**